libsolidity/interface/Version.cpp

```diff
@@ -44,6 +44,34 @@ string const dev::solidity::VersionStringStrict =
 	(string(SOL_VERSION_PRERELEASE).empty() ? "" : "-" + string(SOL_VERSION_PRERELEASE)) +
 	(string(SOL_VERSION_COMMIT).empty() ? "" : "+" + string(SOL_VERSION_COMMIT));
 
+bytes const dev::solidity::VersionCompactBytes = compactBinaryVersion();
+
+bool const dev::solidity::VersionIsRelease = string(SOL_VERSION_PRERELEASE).empty();
+
+bytes dev::solidity::compactBinaryVersion()
+{
+	bytes ret{};
+	size_t i = 0;
+	auto parseDecimal = [&]()
```

Comment:
Code duplication.

Reply:
Turns out `binaryVersion` is not used anymore. How about we just change it to return the 3-byte version?

Reply:
@chriseth what do you think?

Reply:
Sounds good!